client/web/compose/src/components/Common/RecordToolbar.vue

```diff
@@ -27,6 +27,29 @@
         </b-button>
       </div>
 
+      <b-button-group class="ml-auto">
```

Comment:
I think ml-auto can fail if right or left section is hidden. That means it will be centered right if right section doesnt exist. I suggest you use justify-content-between for the row class